WordleGame/WordleGame/Delegates/Keyboard/AlertDelegate.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  AlertDelegate.swift
+//  WordleGame
+//
+//  Created by Alex on 18.05.22.
+//
+
+import Foundation
+
+protocol AlertDelegate {
+    func handleWin(numberOfAttempts: Int)
+    func handleLose()
+    func showAlert(alertText : String, alertMessage : String)
```

Comment:
You don't need this function in protocol now

Reply:
Resolved.)